spec/requests/quiz_taking_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe "Quiz Taking", type: :request do
+  let(:quiz) { Quiz.create!(title: "Sample Quiz", description: "This is a sample quiz") }
+
+  before do
+    quiz.questions.create!(content: "What is the GDS?", correct_answer: "Government Digital Service")
+    quiz.questions.create!(content: "When is Easter?", correct_answer: "April 20th")
+  end
+
+  describe "GET /quizzes/:id/take/:question_index" do
+    it "displays the current question and form" do
+      get take_quiz_path(quiz, 0)
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include("Sample Quiz", "Question 1 of 2", "What is the GDS?", "Submit Answer")
+    end
```

Comment:
I imagine this comment may become redundant but if you do stick with a redirect for a unfound question you'd want to have a test for it.